### PR TITLE
Vidstreaming.io and VidCloud9.com

### PR DESCRIPTION
**Pull request by Gurrman375**
Presences for both sites are stable and working.

**VidCloud9:**
image
image
image
image

**Vidstreaming:**
image
image
image
image

### CODE REVIEW

File: Vidstreaming/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "GURRMAN375D",
+      "id": "446528923535278082"
+    },
+    "url": "vidstreaming.io",
+    "description": {
+      "en": "A Test."
```
Comment (Alanexei) [inline Vidstreaming/dist/metadata.json:8]: Can you change this to something more descriptive?

Comment (Gurrman375) [reply Vidstreaming/dist/metadata.json]: Didn't realize that was still there. It all fixed now.

**Comment by Gurrman375** [comment]
Fixed

File: Vidstreaming/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "service": "Vidstreaming",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/pFcvvz9.png",
+    "thumbnail": "https://i.imgur.com/pFcvvz9.png",
```
Comment (frozaum) [inline Vidstreaming/dist/metadata.json:13]: Can you change the thumbnail to a promotional image or a screenshot of the website?

File: VidCloud9/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "id": "446528923535278082"
+    },
+    "url": "vidcloud9.com",
+    "description": {
+      "en": "VidCloud9 is a great place to find shows you missed or movies your dieing to see."
+    },
+    "service": "VidCloud9",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/AEOuBRB.png",
+    "thumbnail": "https://i.imgur.com/AEOuBRB.png",
```
Comment (frozaum) [inline VidCloud9/dist/metadata.json:13]: same as what i said earlier

Comment (Gurrman375) [reply VidCloud9/dist/metadata.json]: gotcha

File: VidCloud9/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "GURRMAN375D",
+      "id": "446528923535278082"
+    },
+    "url": "vidcloud9.com",
+    "description": {
+      "en": "VidCloud9 is a great place to find shows you missed or movies your dieing to see."
```
Comment (Alanexei) [inline VidCloud9/dist/metadata.json:8]: Grammar mistake, can you please fix that? `dieing`

File: VidCloud9/dist/metadata.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
     },
     "url": "vidcloud9.com",
     "description": {
-      "en": "VidCloud9 is a great place to find shows you missed or movies your dieing to see."
+      "en": "VidCloud9 is a great place to find shows you missed or movies your excited to see."
```
Comment (Bas950) [inline VidCloud9/dist/metadata.json]: You're not your

Comment (Gurrman375) [reply VidCloud9/dist/metadata.json]: Wow that was really bad.

**Pull request merged**